src/common-model/entities.ts

```diff
@@ -39,7 +39,7 @@ export enum ClinicalEntitySchemaNames {
   BIOMARKER = 'biomarker',
 }
 
-export const aliasEntityNames = {
+export const aliasEntityNames: Record<string, string> = {
```

Comment:
it's better to utilize typescript type system than using strings
for example instead of `Record<string, string>` use `Record<EntityName, EntityAlias>` where each one of these is either an enum or literal type `type EntityAlias = 'donor' | 'sampleRegistration' | ....` or  use `keyof ...` 
the point is to avoid Strings in places where arbitrary strings can cause runtime errors

Reply:
Nice I've made these updates, definitely better